packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -25,28 +92,141 @@
   .image-container,
   .select-icon--active {
     @apply h-6 w-6;
+
+    --calcite-internal-chip-padding-start: var(--calcite-spacing-xxs);
+    --calcite-internal-chip-padding-end: var(--calcite-spacing-xxs);
   }
 
-  .container.is-circle {
+  .container.is-circle,
+  .container.is-circle.image--slotted {
     @apply w-8 h-8;
+
+    --calcite-internal-chip-padding-start: var(--calcite-spacing-px);
+    --calcite-internal-chip-padding-end: var(--calcite-spacing-px);
+  }
+
+  .multiple {
+    .select-icon {
+      --calcite-internal-chip-padding-start: var(--calcite-spacing-base);
+      --calcite-internal-chip-padding-end: var(--calcite-spacing-base);
+    }
+
+    &.image--slotted {
+      .select-icon {
+        --calcite-internal-chip-padding-start: var(--calcite-spacing-xxs);
+        --calcite-internal-chip-padding-end: var(--calcite-spacing-sm);
+      }
+
+      &.text--slotted {
+        --calcite-internal-chip-padding-start: var(--calcite-spacing-xxs);
+        --calcite-internal-chip-padding-end: var(--calcite-spacing-xs);
+      }
+    }
+  }
+
+  .container {
+    --calcite-internal-chip-padding-start: var(--calcite-spacing-xs);
+    --calcite-internal-chip-padding-end: var(--calcite-spacing-xs);
+
+    &.image--slotted {
+      --calcite-internal-chip-padding-start: var(--calcite-spacing-xxs);
+      --calcite-internal-chip-padding-end: var(--calcite-spacing-xs);
+
+      &.closable {
+        --calcite-internal-chip-padding-start: var(--calcite-spacing-xxs);
+        --calcite-internal-chip-padding-end: var(--calcite-spacing-xxs);
+      }
+    }
+  }
+
+  .container:not(.is-circle):not(.multiple) {
+    &:not(.image-slotted) .select-icon.select-icon--active {
+      padding-inline-start: 0;
+      padding-inline-end: var(--calcite-spacing-px); // should be 2px, but xxxs doesn't work
```

Comment:
It looks like `--calcite-spacing-xxs` is [not defined](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-design-tokens/src/semantic/space.json). Can you create an issue and update this comment to reference it?

Reply:
It looks like we're using `-base` for 2px and not `xxxs`.